homeassistant/components/climate/strings.json

```
@@ -25,5 +25,120 @@
       "dry": "Dry",
       "fan_only": "Fan only"
     }
+  },
+  "state_attributes": {
+    "_": {
+      "aux_heat": { "name": "Aux heat" },
+      "current_humidity": { "name": "Current humidity" },
+      "current_temperature": { "name": "Current temperature" },
+      "fan_mode": {
+        "name": "Fan mode",
+        "values": {
```

Same suggestion as with the other PR

```suggestion
        "state": {
```